### PR TITLE
[LENG-1722] Expose partnerships config to mobile

### PR DESCRIPTION
**Pull request by reb2020**

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -336,6 +392,54 @@ describe('isRegionAllowed()', function() {
...
+  }) 
+  it('should return krisFlyer', function() {
+    expect(regionModule.getParnershipsByCurrencyCode('SGD', 'luxuryescapes')).to.deep.equal([
+      {
+        hasEarn: true,
+        hasBurn: false,
+        brandName: 'Singapore Airlines',
+        rewardName: 'KrisFlyer Miles',
+        programName: 'Kris flyer',
+        prefix: 'kfp',
```
Comment (mynameisrufus) [inline index.test.js]: Can we change `prefix` to `code`?

How about:

Qantas Frequent Flyer

```json
{
  "code": "qff"
}
```

Kris Flyer:

```json
{
  "code": "kfp"
}
```

File: __mocks__/currency-data.js [hunk 2]
```diff
@@ -33,6 +33,14 @@ module.exports = {
           'maple_syrup_eh',
         ],
       },
+      SGD: {
+        payment_methods: [
+          'le_credit',
+          'stripe',
+          'giftcard',
+          'krisFlyer',
```
Comment (mynameisrufus) [inline __mocks__/currency-data.js]: This is camel but everything else is snake

File: index.test.js (hunk 1 above)
Comment (marhalpert) [reply index.test.js]: I will change the customer to use code as well

Comment (marhalpert) [reply index.test.js]: actually, all the new auth stuff already uses the prefix

**Pull request merged**